pom.xml

```diff
@@ -82,6 +82,8 @@
     <hadoop.version>3.0.3</hadoop.version>
     <hbase.thirdparty.version>2.2.1</hbase.thirdparty.version>
 
+    <phoenix.thirdparty.version>1.0.0-SNAPSHOT</phoenix.thirdparty.version>
```

Comment:
Noting that these are pulling snapshots that someone deploys by hand. We need to prioritize a release of these repo's (thirdparty as well as the tephra and omid release) in some shape/form. Doesn't matter to me if we slap a "-beta" suffix onto any of them, just depending on SNAPSHOT releases long term is going to create problems for us.

Reply:
Omid and Tephra have the same SDNAPSHOT issue.

As soon as we get this patch in, and it has had a week or so to prove not having caused major regressions, I'll start the release discussion for all dependee subprojects (thirdparty, omid, tephra) in preparation of 5.1.

I did not want to release those, until we have had a chance to test their full integration with Phoenix.

Reply:
Sounds good! You have a plan and that's good enough for me.